packages/dd-trace/src/debugger/devtools_client/state.js

```diff
@@ -32,8 +32,17 @@ module.exports = {
       .sort(([a], [b]) => a.length - b.length)[0]
   },
 
-  getScriptUrlFromId (id) {
```

Comment:
`getScriptUrlFromId` was only used for formatting the stack, which is now done by `getStackFromCallFrames`